src/normalize-options.ts

```diff
@@ -100,10 +105,42 @@ export async function normalizeOptions(raw: VersionBumpOptions): Promise<Normali
   else if (raw.commit || tag || push)
     commit = { all, noVerify, message: 'chore: release v' }
 
-  const files = await fg(
-    raw.files?.length
+  if (recursive) {
+    if (raw.files?.length) {
+      console.log(c.yellow('The --recursive option is ignored when files are specified'))
```

Comment:
We better move this console log out of this function especially if you want to use programmaically

Reply:
Just moved the console log back to cli